mem/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for mem 1.1.0
+// Project: https://github.com/sindresorhus/mem
+// Definitions by: Sam Verschueren <https://github.com/SamVerschueren>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module "mem" {
+	interface Options {
+		maxAge?: number;
+		cacheKey?: (...args: any[]) => string;
+		cache?: any;
+	}
+
+	function mem<T>(fn: (...args: any[]) => T, options?: Options): (...args: any[]) => T;
+
+	namespace mem { }
```

Comment:
We don't recommend doing this trick; see [here](https://stackoverflow.com/questions/39415661/why-cant-i-import-a-class-or-function-with-import-as-x-from-y).